plugins/search/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -80,7 +86,7 @@ const CheckboxFilter = ({
 
   return (
     <FormControl
-      className={className}
+      className={className || classes.filter}
```

Comment:
This an interesting way to provide backwards compatibility for the way it's rendered in the default App search page, but I guess if you were previously using it without providing a className, you'd suddenly have extra margin in your filter?

Reply:
hmm good point, see my comment above, i'll do the same here!